libs/blocks/merch-card/merch-card.js

```diff
@@ -29,7 +29,7 @@ const CARD_TYPES = [
 const CARD_SIZES = ['wide', 'super-wide'];
 
 const TEXT_STYLES = {
-  H5: 'detail-m',
+  H5: 'promo-text',
```

Comment:
I wonder if this change doesn't impact other cards.

Reply:
it does! for all desktop+mobile texts. Please look at the ticket there is a reference on us removing that detail-m thing